cmd/kubeapps-apis/docs/kubeapps-apis.swagger.json

```diff
@@ -3565,6 +3565,69 @@
         ]
       }
     },
+    "/plugins/resources/v1alpha1/c/{context.cluster}/can-i": {
```

Comment:
This API docs should also be (manually) copied to the `dashboard/src/public/openapi.yaml` file, but we can do it once we remove the kubeapps deployment entirely.